config.toml

```diff
@@ -58,6 +58,7 @@ If your main cluster fails, you can bring up the app in the DR cluster quickly w
 
 [params.social]
 github = "https://github.com/longhorn/longhorn"
+slack = "https://slack.cncf.io"
```

Comment:
This won't redirect directly to Longhorn channel right? I am wondering if there is a better way.

Reply:
This would work https://cloud-native.slack.com/messages/longhorn